src/wp-admin/includes/class-custom-background.php

```diff
@@ -239,21 +239,23 @@ public function admin_page() {
 <div class="wrap" id="custom-background">
 <h1><?php _e( 'Custom Background' ); ?></h1>
 
-		<?php if ( current_user_can( 'customize' ) ) { ?>
-<div class="notice notice-info hide-if-no-customize">
-	<p>
-			<?php
-			printf(
-				/* translators: %s: URL to background image configuration in Customizer. */
-				__( 'You can now manage and live-preview Custom Backgrounds in the <a href="%s">Customizer</a>.' ),
-				admin_url( 'customize.php?autofocus[control]=background_image' )
+		<?php
+		if ( current_user_can( 'customize' ) ) {
+			wp_admin_notice(
```

Comment:
I pretty much always assigned longer messages to a variable; we should possibly be consistent about this. My intention was to reduce indentation for better message readability, but I can go either way. Thoughts?

Reply:
I'm happy either way, we just need to be careful of variable naming